src/js/contentscripts/socialwidgets.js

```diff
@@ -288,6 +279,64 @@ function replaceSubsequentTrackerButtonsHelper(trackerDomain) {
   });
 }
 
+function createReplacementWidget(button, buttonToReplace, trackerUrls) {
+  let widgetFrame = document.createElement('iframe');
+
+  // widget replacement frame styles
+  let styleAttrs = [
+    "border: 1px solid #ec9329",
+    "padding: 10px",
+    "width:" + buttonToReplace.clientWidth + "px",
```

Comment:
Any way we can copy the width/height from the old element, so that percentages carry over too? As is, the replacement widget won't react to page resizes well.

Reply:
e.g. `buttonToReplace.style.width`?

Reply:
I know what you mean, but I'm not sure there is a simple way of achieving this. I think best left for a potential future improvement.

Reply:
fine with me, this is another nit.